addons/core/translations/resources/en-us.yaml

```diff
@@ -435,6 +435,9 @@ storage-bucket:
     dynamic:
       title: Dynamic
       description: Specify an Assume Role used to generate credentials to use with this AWS bucket
+  plugin-types:
+    aws: Aws
```

Comment:
I believe this should be `AWS`

Reply:
Actually, figma shows `Amazon S3`

Reply:
@DhariniJeeva 
You are totally right, changing it! thanks for the catch! 😉